aipython/stripsCSPPlanner.py

```diff
@@ -25,42 +25,84 @@ def __init__(self, planning_problem, horizon=2):
         prob_domain = planning_problem.prob_domain
         initial_state = planning_problem.initial_state
         goal = planning_problem.goal
+
+        old_pos = planning_problem.positions
+
+        maxwidth = max([x for x, y in old_pos.values()])
```

Comment:
It seems that you need to deal with the empty array


<img width="841" alt="image" src="https://user-images.githubusercontent.com/39361141/64058542-78c5b580-cb61-11e9-95ce-4a4de99e50c3.png">